packages/dd-trace/src/appsec/reporter.js

```diff
@@ -48,6 +46,14 @@ const IDENTIFICATION_HEADERS_MAP = mapHeaderAndTags([
   'x-sigsci-requestid',
   'x-sigsci-tags',
   'akamai-user-risk'
+]
+
+// these request headers are always collected - it breaks the expected spec orders
+const MANDATORY_REQUEST_HEADERS_MAP = mapHeaderAndTags([
```

Comment:
maybe a bit better to have this be called `REQUEST_HEADERS_MAP` and the other one `ATTACK_HEADERS_MAP` ?